fhir-operation-bulkdata/src/main/java/com/ibm/watson/health/fhir/operation/bullkdata/client/BulkDataClient.java

```diff
@@ -179,7 +188,7 @@ public String submit(MediaType outputFormat, Instant since, List<String> types,
         // From the response
         String jobId = Integer.toString(response.getInstanceId());
 
-        return "/fhir-server/api/v4/$export-status?job=" + jobId;
+        return "/wh-fhir-dev/api/v4/$export-status?job=" + jobId;
```

Comment:
will merge for now, but this needs to get updated to more inteligently set the export-status url